yarn.lock

```diff
@@ -6341,6 +6341,10 @@ react-dom@16.4.2:
     object-assign "^4.1.1"
     prop-types "^15.6.0"
 
+react-favicon@0.0.14:
```

Comment:
`yarn.lock` needs updating (I thought this should make CI fail, due to the use of `yarn install --frozen-lockfile`, but apparently that only catches missing *additions* not missing *removals* by design - https://github.com/yarnpkg/yarn/issues/5840#issuecomment-414581160)